src/server/protocol.ts

```diff
@@ -955,6 +955,16 @@ namespace ts.server.protocol {
         file: string;
     }
 
+    export interface JSDocTagInfo {
+        name: string;

```

Comment:
Are these meant to be duplicated with the types in `types.ts`?

Reply:
Yes, the language service ones in `types.ts` use character offsets and the ones here use line+character offsets.